app/helpers/events_helper.rb

```diff
@@ -47,6 +47,52 @@ def ical_from_collection(events)
     cal.to_ical
   end
 
+  def rss_from_collection(events)
+    require 'rss'
```

Comment:
I would move this to the top of the file just so any loading issues are apparent immediately, rather than when the method is called